develop/learning-paths/migration/articles/xx-theme/04-updating-the-resources-importer.markdown

```diff
@@ -0,0 +1,365 @@
+    `/src/resources-importer/journal/articles/Basic Web Content/` directory and
+    replace the contents with the markup below:
+    
+    <?xml version="1.0"?>
+    
+    <root available-locales="en_US" default-locale="en_US">
+            <dynamic-element name="content" type="text_area" 
+            index-type="keyword" index="0">
+                    <dynamic-content language-id="en_US">
+                            <![CDATA[
+                                    <p>Thanks for choosing to stay at the 
+                                    Liferay Lunar Resort! Please fill out the 
+                                    form below to book your stay. We know you 
+                                    have a choice in where to stay on the Moon... 
+                                    oh wait no you don't. Thanks for picking us 
+                                    anyways. We'll see you soon on the Moon!</p>
+    <form role="form-horizontal">
+            <fieldset>
+              <legend>Reservation Form</legend>
+              <div class="form-group">
+                  <label for="inputName">Name</label>
+                  <input type="text" id="inputName" class="form-control" 
+                  placeholder="Enter your Name here" required="required">
+              </div>
+              <div class="form-group">
+                  <label for="inputEmail">Email</label>
+                  <input type="email" id="inputEmail" class="form-control" 
+                  placeholder="Enter your E-Mail here" required="required">
+              </div>
+              <div class="form-group">
+                      <button type="submit" class="btn btn-primary">Submit
+                      </button>
+              </div>
+            </fieldset>
+    </form>
+    
+    <p style="padding-bottom:25px;">Thanks again for booking with Liferay. When 
+    you book with Liferay, you remember your stay. Please take a moment to fill 
+    out our guestbook below.</p>
+                            ]]>
+                    </dynamic-content>
+            </dynamic-element>
+    </root>
+
+The `control-group` classes were updated to `form-group` classes. The 
+The `control-label` classes were removed from the `<label>` elements. The 
+`<div class=""controls>` elements were removed. A `form-control` class was added 
+to each `<input>` element. Finally, the `btn-primary` class was added to the 
+submit button for the form, to give it more emphasis and contrast.
+
+Now that the basic web content is updated, all that is left to update is the
+carousel article's structure and template.
+
+### Updating the Carousel's Structure and Template
+
+The carousel article uses a more advanced structure and template than the basic 
+web content. Luckily, the carousel article is already XML, so it doesn't need 
+updated.
+
+Follow these steps to migrate the 6.2 carousel structure and template to 
+Liferay 7:
+
+Liferay 6.2 used AlloyUI 2.5.x, Liferay 7 uses Alloy UI 3.0. There has been a
```

Comment:
@Robert-Frampton I've made updates per your comments. Are there any other comments or edits, or is this good to go?